src/server/src/app.ts

```diff
@@ -1,11 +1,32 @@
-import 'dotenv/config';
-import express from 'express';
-import path from 'path';
-import db from './db';
+import "dotenv/config";
```

Comment:
Probably need to add a prettierrc config file to fix these issues. I can add them in my latest changes!